build.py

```diff
@@ -1171,9 +1171,10 @@ def create_dockerfile_windows(ddir, dockerfile_name, argmap, backends,
 
 def container_build(images, backends, repoagents, endpoints):
     # The cmake, build and install directories within the container.
-    build_dir = os.path.join(FLAGS.tmp_dir, 'tritonbuild')
+    build_dir = os.path.join(FLAGS.tmp_dir, 'tritonbuild').replace("\\", "/")
```

Comment:
Add comment on why added the "replace", something about "\\" is added for Windows path but Docker doesn't accept that.